admin/base/settings/defaults.py

```diff
@@ -82,4 +96,15 @@
 ADMIN_BASE = 'admin/'
 STATIC_URL = '{}static/'.format(ADMIN_BASE)
 
+STATIC_ROOT = os.path.join(os.path.dirname(BASE_DIR), 'static_root')
+
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, 'static'),
+)
+
 LANGUAGE_CODE = 'en-us'
+
+WEBPACK_LOADER = {
+    'BUNDLE_DIR_NAME': 'public/js/',
```

Comment:
I would suggest putting these two lines in a dictionary with a keyword `DEFAULT`, otherwise this doesn't load properly in my environment.